index.html

```diff
@@ -36,8 +37,32 @@
         </div>
       </div>
 
+        <!-- <div class="pop-up-mobile" id="popupmobile">
+        <div class="pop-up-header" id="popupclosemobile">
+          <h2>Multi Post Stories</h2>
+          <span class="close">&times;</span>
+        </div>
+        <div>
+          <ul class="pop-up-tags">
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+          </ul>
+        </div>
+        <div class="mobile-popup-img">
+        <img src="Images\Mobile-popup.png" alt="popup-img.png">
+        </div>
+        <div class="popup-description">
+          <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p>
+        </div>
+        <div class="popup-buttons">
+          <button id="see-live">See Live <img src="Images\mob-popup.png" alt="img.png"></button>
+          <button id="see-source">See Source <img src="Images\pop-icon-github.png" alt="img.png"></button>
+        </div>
+    </div> -->
```

Comment:
It's highly recommended to keep your repository as clean as possible, and removing the unused commented code will help you keep it clean. Kindly make sure to remove it.